tests/php/PDepend/Bugs/AbstractRegressionTestCase.php

```diff
@@ -99,6 +99,7 @@ public function parseTestCaseSource(string $testCase, bool $ignoreAnnotations =
      * Returns the source file for the given test case.
      *
      * @param string $testCase The qualified test case name.
+     * @throws Exception
```

Comment:
It's a good practice to document when such an exception is thrown.

```suggestion
     * @throws Exception When key `1` in `$match` is not set.
```

```suggestion
     * @throws Exception When test case ID can't be found.
```

Reply:
Yeah, I was thinking this would be left as a todo or since I'm only documenting the case and would have to investigate in order to be able to write a propper comment.